### PR TITLE
Deployment of .... actually a ton of things

### PR DESCRIPTION
**Pull request by sphinxrave**

### CODE REVIEW

File: src/views/Watch.vue [hunk 1]
```diff
@@ -175,7 +176,7 @@ import WatchHighlights from "@/components/watch/WatchHighlights.vue";
 import WatchToolBar from "@/components/watch/WatchToolbar.vue";
 import WatchComments from "@/components/watch/WatchComments.vue";
 import UploadScript from "@/components/tlscriptmanager/UploadScript.vue";
-import { decodeHTMLEntities, syncState } from "@/utils/functions";
+import { decodeHTMLEntities, syncState, getYTLangFromState} from "@/utils/functions";
```
Comment (LiviaMedeiros) [inline src/views/Watch.vue:179]:
```suggestion
import { decodeHTMLEntities, syncState, getYTLangFromState } from "@/utils/functions";
```

File: src/utils/consts.js [hunk 2]
```diff
@@ -136,6 +136,15 @@ export const TL_LANGS = Object.freeze([
 //     MISSING: "missing",
 // };
 
+
```
Comment (LiviaMedeiros) [inline src/utils/consts.js:139]:
```suggestion
```

File: src/utils/consts.js [hunk 3]
```diff
@@ -136,6 +136,15 @@ export const TL_LANGS = Object.freeze([
...
+// Language code conversion for YouTube frame
+export const langConversion = Object.freeze({
+    "lol-PEKO": "en",
+    "lol-UWU": "en",
+    "zh": "zh-Hant",
+    "zh-CN": "zh-Hans"
```
Comment (LiviaMedeiros) [inline src/utils/consts.js:145]:
```suggestion
    "zh-CN": "zh-Hans",
```

File: src/utils/functions.js [hunk 4]
```diff
@@ -75,6 +76,11 @@ export function getLang(weblang) {
     return "en";
 }
 
+export function getYTLangFromState(state) {
+    const lang = state.settings.lang;
```
Comment (LiviaMedeiros) [inline src/utils/functions.js:80]:
```suggestion
    const { lang } = state.settings;
```

**Review by LiviaMedeiros — COMMENTED** [review]
beep bop am linter

**Review by NeloBlivion — APPROVED** [review]
lgtm

**Pull request merged**